### PR TITLE
No longer saving attention mask, option for TTS percentages

### PR DESCRIPTION
**Pull request by KPHippe**
No longer saving the attention mask leads to marginal improvements in storage. TTS allows you to specify exact percentages for your splits

### CODE REVIEW

**Comment by KPHippe** [comment]
Do not merge yet... need to update H5 datasets to read in the `attention_mask` if it exists or create it if it doesn't. Not there yet

**Comment by KPHippe** [comment]
Going into `develop_resolve` branch, sorry I messed up 😬.
